packages/e2e-tests/specs/blocks/group.test.js

```diff
@@ -25,7 +24,21 @@ describe( 'Group', () => {
 		await clickBlockAppender();
```

Comment:
It wasn't introduced here, so not necessary to change, but I find that testing the block inserter and the slash inserter as distinct tasks is not something which should be done in tests for individual blocks, but rather specific to those inserter interactions. This could save us a fair amount of effort (and build runtime) to avoid running them for every combination of blocks.